packages/blast-core/README.md

```diff
@@ -337,6 +338,69 @@ async function main () {
   // ...
 ```
 
+---
+## Creating a custom task
+
+Cudos Blast allows the creation of custom tasks that can easily run commonly used operations or help manage your workflow.
+To add a new task you have to set it up in the `blast.config.js` file.
```

Comment:
I think this sentence is obsolete. Later the guide will tell the user to add the task in the config anyway